generators/python/src/fern_python/generators/sdk/client_generator/endpoint_function_generator.py

```diff
@@ -1081,6 +1082,18 @@ def write_ternary(writer: AST.NodeWriter) -> None:
 
                 reference = AST.Expression(AST.CodeWriter(write_ternary))
 
+        elif self._context.custom_config.pydantic_config.use_typeddict_requests and FernTypedDict.can_tr_be_typeddict(
+            query_parameter.value_type, self._context.get_types()
+        ):
+            # We don't need any optional wrappings for the coercion here.
```

Comment:
what exactly is this doing ?

Reply:
This is unwrapping the optional off the type, so that we pass the underlying type to the serialization method, as opposed to the optional type

Reply:
Gotcha, will check the seed snapshot